File: submissions/sm_029_sakshi/week_3/day_3/session_2/average_divisible.js
function avgdiv(limit,number)
{
    var sum=0
    var count=0
    for(var i=0;i<=limit;i++)
    {
        if(i%number==0)
        {
            sum=sum+i
            count=count+1
          //console.log(sum)
           //console.log(count)
        }
      // 

    }
    return sum/count
}
avgdiv(30,4)
console.log(avgdiv(30,4))

File: submissions/sm_105_ashish/week_13/day_4/session_1/average_divisible.py
limit = int(input('Please enter the limit :'))
number = int(input('Please enter the number :'))
sum = 0
count = 0
for x in range(0, limit+1,number):
    if x%number == 0:
        sum = sum + x
        count = count + 1
print(sum/(count-1))    